game/src/sandbox/dashboards/summaries.rs

```diff
@@ -260,18 +260,20 @@ fn contingency_table(ctx: &mut EventCtx, app: &App, filter: &Filter) -> Widget {
     if points.is_empty() {
         return Widget::nothing();
     }
+
+    // bucket by trip duration _before_ changes
     let num_buckets = 10;
     let (_, endpts) = points
         .iter()
-        .map(|(b, a)| a.max(b))
+        .map(|(b, _a)| b)
```

Comment:
Since we were grouping by  the *before* duration, our num_buckets should be based only on the *before* durations.